packages/context/README.md

```diff
@@ -306,8 +308,99 @@ instanceChanges.unsubscribe('myObjectId', callback);
 #### `useSignalRConnection()`
 
 > **Deprecated**
-> Use of `useSignalRConnection()` has been deprecated. Use of this will not cause an error, but the
-> widget will not receive any notifications. Use [useNotification](#usenofitication) instead.
+> This has been deprecated in favor of [useNotification](#usenofitication).
+
+Hook used to obtain an instanceChanges instance of `SignalRConnection` and a documentChanges instance of `SignalRConnection`.
+
+##### `documentChanges.subscribe('{objectId}/{instanceId}', (data: DocumentChange[]]) => {})`
+
+> **Deprecated**
+> This has been deprecated in favor of [useNotification](#usenofitication).
+
+Subscribe to the specified object instance document changes.
+
+```javascript
+const { documentChanges } = useSignalRConnection();
+
+documentChanges.subscribe('myObjectId/myInstanceId', (data) => {
+    console.log(data);
+});
+```
+
+The data provided to the callback will be an array of `DocumentChange` which contains the
+following data:
+
+-   `objectId`
+    -   Object describing the instance associated with the updated document.
+-   `instanceId`
+    -   Instance that the updated document is associated with.
+-   `documentId`
+    -   Document that was updated.
+-   `type`
+    -   The type of update. Possible values are `BlobCreated`, `BlobDeleted`, and `BlobMetadataUpdated`.
+
+##### `documentChanges.unsubscribe('{objectId}/{instanceId}', (data: DocumentChange[]) => {})`
+
+> **Deprecated**
+> This has been deprecated in favor of [useNotification](#usenofitication).
+
+Unsubscribe to the specified object instance document changes.
+
+Callback function is optional.
+If callback function is defined, you must pass the exact same Function instance as was previously passed to `documentChanges.subscribe`.
```

Comment:
@actong not sure if this was the same before as well, but it seems to me this sentence needs to be highlighted more / marked important. What do you think?

Reply:
The PR diff makes it hard to see, but the content is the same as before (other than I added the **Deprecated** notices), I just moved `useSignalRConnection` below `useNotification` (didn't want the deprecated version to come first).

As for marking it more important, I don't have any strong feelings one way or the other.  I think it's pretty standard that when unsubscribing/removing a listener, you'd have to pass the actual listener you want to unsubscribe.